packages/database/src/datasources/index.ts

```diff
@@ -230,6 +250,12 @@ export async function getDatabaseURL(
       )
       return `mysql+mysqldb://${ds.data.username}:${password}@${ds.data.host}:${ds.data.port}/${ds.data.database}?ssl_mode=REQUIRED`
     }
+    case 'sqlserver': {
+      const password = encodeURIComponent(
+        await getDatasourcePassword(ds, encryptionKey)
+      )
+      return `mssql+pymssql://${ds.data.username}:${password}@${ds.data.host}:${ds.data.port}/${ds.data.database}`
```

Comment:
Remember to change this after changing the lib - see note above about picking an MIT/BSD/Apache (or equivalent) license

Reply:
This is fine, double checked.